plugins/MumblePlugin_v_1_2_x.h

```diff
@@ -0,0 +1,477 @@
+// Copyright 2021-2022 The Mumble Developers. All rights reserved.
```

Comment:
Is there any change to the plugin interface (the callbacks)? If not, we should leave the plugin functions at version 1.1.x. The API and the plugin callback interface essentially have an independent versioning scheme and don't have to match up.

Reply:
No it is not necessary. You linked to this file specifically in your original post and so I thought it was important to increase the API version here, too. I removed the file in the latest commit

Reply:
Ah whoops - then I linked the wrong file :facepalm: